blocks/card-list-block/features/card-list/default.jsx

```diff
@@ -4,10 +4,25 @@ import PropTypes from "@arc-fusion/prop-types";
 import { useContent } from "fusion:content";
 import { useFusionContext } from "fusion:context";
 import getProperties from "fusion:properties";
-import { Image, LazyLoad, isServerSide } from "@wpmedia/engine-theme-sdk";
+import getTranslatedPhrases from "fusion:intl";
+import { LazyLoad, localizeDate } from "@wpmedia/engine-theme-sdk";
 import { extractResizedParams, extractImageFromStory } from "@wpmedia/resizer-image-block";
```

Comment:
We need to remove this block from being imported

Reply:
oh my bad. will work on this

Reply:
updated -- missed this in one of my stashes